### PR TITLE
Use embedded chain spec for turing-live

### PR DESCRIPTION
**Pull request by nvengal**
This change allows community collators to run turing without managing chainspec files or knowing bootnode address. Specifying just the turing chain is enough for the node to use the embedded parachain spec and find the corresponding relaychain spec for kusama.

Since neumann is running on a relay chain hosted by us, we need to specify both the chain and relay chain-
```
❯ ./target/release/oak-collator --chain neumann -- --chain neumann-relay
```

Turing chain started syncing fine when I tested locally. (I didn't sync the full chain or ensure block production)
```
❯ ./target/release/oak-collator --tmp --chain turing
2022-05-13 15:08:56 OAK Collator
2022-05-13 15:08:56 ✌️  version 1.2.8-3befd33-aarch64-macos
2022-05-13 15:08:56 ❤️  by OAK Developement Team, 2021-2022
2022-05-13 15:08:56 📋 Chain specification: Turing Network
2022-05-13 15:08:56 🏷  Node name: sophisticated-yoke-7577
2022-05-13 15:08:56 👤 Role: FULL
2022-05-13 15:08:56 💾 Database: RocksDb at /var/folders/ts/2ld7z2550xs_290tp82ctz580000gn/T/substrateZMImDS/chains/turing/db/full
2022-05-13 15:08:56 ⛓  Native runtime: turing-280 (turing-1.tx7.au1)
2022-05-13 15:08:56 Parachain id: Id(2114)
2022-05-13 15:08:56 Parachain Account: 5Ec4AhNzVAVwDR54oqgWQ5YSoPbxXathSrQqTmqkF5o7xe6y
2022-05-13 15:08:56 Parachain genesis state: "0x000000000000000000000000000000000000000000000000000000000000000000d62485649430ebd0e40c69274b3819776921be357f66ce57223f180a6fbb4f8503170a2e7597b7b7e3d84c05391d139a62b157e78786d8c082f29dcf4c11131400"
2022-05-13 15:08:56 Is collating: no
2022-05-13 15:08:56 [Parachain] 🔨 Initializing Genesis block/state (state: 0xd624…4f85, header-hash: 0x0f62…535d)
2022-05-13 15:08:57 [Relaychain] 🔨 Initializing Genesis block/state (state: 0xb000…ef6b, header-hash: 0xb0a8…dafe)
2022-05-13 15:08:57 [Relaychain] 👴 Loading GRANDPA authority set from genesis on what appears to be first startup.
2022-05-13 15:08:57 [Relaychain] 👶 Creating empty BABE epoch changes on what appears to be first startup.
2022-05-13 15:08:57 [Relaychain] 🏷  Local node identity is: 12D3KooWSohLBvdoJp2mnu4V94L2TchTv3dPmmVGLg9rpLVWoMgn
2022-05-13 15:08:57 [Relaychain] 📦 Highest known block at #0
2022-05-13 15:08:57 [Relaychain] 〽️ Prometheus exporter started at 127.0.0.1:9616
2022-05-13 15:08:57 [Relaychain] Listening for new connections on 127.0.0.1:9945.
2022-05-13 15:08:57 [Parachain] 🏷  Local node identity is: 12D3KooWBGXddniz4i44AamYJ9hSb6X2ZJwLcFRLrkEgDfGimhPo
2022-05-13 15:08:57 [Parachain] 📦 Highest known block at #0
2022-05-13 15:08:57 [Parachain] 〽️ Prometheus exporter started at 127.0.0.1:9615
2022-05-13 15:08:57 [Parachain] Listening for new connections on 127.0.0.1:9944.
2022-05-13 15:08:57 [Relaychain] Failed to fetch runtime API data for job err=Execution { runtime_api_name: "validator_groups", source: Application(Execution(Other("Exported method ParachainHost_validator_groups is not found"))) }
2022-05-13 15:08:57 [Relaychain] Failed to fetch basics from runtime API relay_parent=0xb0a8d493285c2df73290dfb7e61f870f17b41801197a149ca93654499ea3dafe err=Execution { runtime_api_name: "session_index_for_child", source: Application(Execution(Other("Exported method ParachainHost_session_index_for_child is not found"))) }
2022-05-13 15:08:57 [Relaychain] Our view contains 0xb0a8…dafe, but not in active heads added=0xb0a8…dafe
2022-05-13 15:08:57 [Relaychain] error=Sender(Runtime(RuntimeRequest(Execution { runtime_api_name: "session_index_for_child", source: Application(Execution(Other("Exported method ParachainHost_session_index_for_child is not found"))) }))) ctx="on FromOverseer"
2022-05-13 15:08:57 [Relaychain] job finished with an error job="bitfield-signing-job" parent_hash=0xb0a8…dafe err=Util(RuntimeApi(Execution { runtime_api_name: "session_index_for_child", source: Application(Execution(Other("Exported method ParachainHost_session_index_for_child is not found"))) }))
2022-05-13 15:08:57 [Relaychain] error=RollingSessionWindow(SessionsUnavailable { kind: RuntimeApi(Execution { runtime_api_name: "session_index_for_child", source: Application(Execution(Other("Exported method ParachainHost_session_index_for_child is not found"))) }), info: None })
2022-05-13 15:08:58 [Relaychain] 🔍 Discovered new external address for our node: /ip4/97.120.124.235/tcp/30334/ws/p2p/12D3KooWSohLBvdoJp2mnu4V94L2TchTv3dPmmVGLg9rpLVWoMgn
2022-05-13 15:08:58 [Parachain] 🔍 Discovered new external address for our node: /ip4/97.120.124.235/tcp/30333/ws/p2p/12D3KooWBGXddniz4i44AamYJ9hSb6X2ZJwLcFRLrkEgDfGimhPo
2022-05-13 15:08:58 [Relaychain] Received msg before first active leaves update. This is not expected - message will be dropped. msg=ActiveDisputes(Sender { complete: false })
2022-05-13 15:08:58 [Relaychain] error=Sender(AskActiveDisputesCanceled) ctx="on FromOverseer"
2022-05-13 15:09:00 [Parachain] assembling new collators for new session 2 at #1800
2022-05-13 15:09:02 [Relaychain] ⚙️  Syncing, target=#12671905 (21 peers), best: #7680 (0x8862…b1be), finalized #7168 (0x72a9…8d8c), ⬇ 658.5kiB/s ⬆ 47.2kiB/s
2022-05-13 15:09:02 [Parachain] ⚙️  Syncing, target=#162806 (24 peers), best: #3058 (0xa543…16c5), finalized #0 (0x0f62…535d), ⬇ 3.7MiB/s ⬆ 15.4kiB/s
```

### CODE REVIEW

File: node/res/turing.json [hunk 1]
```diff
@@ -15,7 +19,7 @@
     "tokenDecimals": 10,
     "tokenSymbol": "TUR"
   },
-  "relay_chain": "ksmcc3",
```
Comment (nvengal) [inline node/res/turing.json:18]: I'm not sure what `ksmcc3` is but setting the relay chain as kusama allows the node to infer the relay chain spec without explicitly passing a file.

File: node/src/chain_spec/turing.rs [hunk 2]
```diff
@@ -199,112 +198,8 @@ pub fn turing_staging() -> ChainSpec {
...
-		// Properties
-		Some(properties),
-		// Extensions
-		Extensions {
-			relay_chain: TURING_RELAY_CHAIN.into(), // You MUST set this to the correct network!
-			para_id: live_para_id,
-		},
-	)
+pub fn turing_live() -> Result<DummyChainSpec, String> {
+	DummyChainSpec::from_json_bytes(&include_bytes!("../../res/turing.json")[..])
```
Comment (nvengal) [inline node/src/chain_spec/turing.rs:202]: Borrowed this pattern from polkadot- https://github.com/paritytech/polkadot/blob/d5f16df743931d535441f28c605e619c16b971a3/node/service/src/chain_spec.rs#L146

File: node/res/turing.json [hunk 3]
```diff
@@ -2,7 +2,11 @@
   "name": "Turing Network",
   "id": "turing",
   "chainType": "Live",
-  "bootNodes": [],
+  "bootNodes": [
```
Comment (irsal) [inline node/res/turing.json:5]: one more from blockdaemon
`/dns4/oak-turing-bootnode-01.bdnodes.net/tcp/30333/p2p/12D3KooWL9phojBkHLWaATRmzjh4YN6NKtdGPufNpPegHF3NDios`

**Review by irsal — APPROVED** [review]
one comment

File: node/res/turing.json (hunk 1 above)
Comment (irsal) [reply node/res/turing.json:18]: we got it from here: https://github.com/paritytech/polkadot/blob/master/node/service/res/kusama.json

Comment (nvengal) [reply node/res/turing.json:18]: Interesting. I'm 99% sure, `kusama` is what we actually want here, to be able to defer to polkadot for the chain spec.

File: node/res/turing.json (hunk 3 above)
Comment (nvengal) [reply node/res/turing.json:5]: Added

**Comment by nvengal** [comment]
Removed some files that I think are unused here- https://github.com/OAK-Foundation/OAK-blockchain/pull/142/commits/f8fe3d1d296a3029941ac8f673c45ad645d884e7
Would be good to give it a second look.

File: node/res/turing.json (hunk 1 above)
Comment (irsal) [reply node/res/turing.json:18]: if it works w/ our chain, that's fine. We'll hear about issues from the collators

**Review by irsal — APPROVED** [review]
second look ...looks good

**Pull request merged**